### ISSUE TITLE
Scope companion dirs from program_name/arg_values[0]

### ISSUE THREAD

**Issue by ramity** [ramity/iris#26]

**Comment by ramity** [comment]
Better to leave the current implementation to handle windows. I was thinking it'd be smart just to process the program_name invocation and remove the "iris" characters to build the keys and identities paths, but doing so won't be something I can mimic for windows envs.

The way that the program currently does it isn't too great either and requires execution of the iris binary from within the containing dir. For instance, let's say I execute it from outside the build dir by running the following:
```
./build/iris
```
The values of `current_path`, `default_keys_path`, and `default_identities_path` are respectively:
```
/root/iris
/root/iris/keys
/root/iris/identities
```
In other words, the current approach doesn't consider that a user may be attempting to call the iris binary from outside the directory it resides in, and that needs to be taken into account when defining the default dirs.
